examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,110 @@
+# Centralized logging for multi-tenant applications on Amazon EKS
+
+In this example, we will showcase how to build a centralized logging solution for multi-tenant environments where applications that belong to different teams or customers run in a shared Amazon EKS cluster.
+
+## Components
+
+[Fluent Bit](https://fluentbit.io/) DaemonSet to collect, modify, and enrich logs from applications, and publish the logs to Amazon Managed Streaming for Kafka (Amazon MSK).
+
+[Amazon MSK](https://aws.amazon.com/msk/) to forward log events to various destination and as a buffering layer to avoid indexing pressure in Amazon OpenSearch. This layer will provide tenant isolation and improve resilience of the solution.
```

Comment:
I like this new intro from the POV of selling this tutorial... but why did you remove the terminology section explain what words like tenant mean? I liked having that section...